packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -116,7 +120,15 @@ export let visitor = {
     // otherwise `arguments` will be remapped in arrow functions
     argsId._shadowedFunctionLiteral = path;
 
-    // support patterns
+    function optimiseLoadStatement(parent, offset) {
+      let newExpr = loadRest({
```

Comment:
no need to assign it to a variable to only return it in the next statement, merge this into a single statement pelase